monkey/infection_monkey/send_all_events_to_island.py

```diff
@@ -0,0 +1,112 @@
+import logging
+import queue
+import threading
+from typing import Set
+
+import requests
+
+# TODO: shouldn't leak implementation information; can we do this some other way?
+from pubsub import pub
```

Comment:
You just don't need it. Firstly, `topic` in `__call__()` will always be "all_events_topic", so the log message isn't useful. Secondly, you can get the event type from the event itself with `type(event).__name__`.